### PR TITLE
AAE-10058 fix test docker images removal

### PR DESCRIPTION
**Pull request by atchertchian**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -382,6 +382,13 @@ jobs:
                 make delete
             fi
 
+  delete-test-images:
```
Comment (atchertchian) [inline .github/workflows/main.yml:385]: doing this in another step does not work ok since the checkout is done again -> specific set version is lost

Comment (balsarori) [reply .github/workflows/main.yml:385]: yeah better to move the delete images to their own step. Also, we can skip that step until we figure out why the delete image is failing.

Comment (atchertchian) [reply .github/workflows/main.yml:385]: I'm now passing the version, hopefully that should fix the logic.
Now the deletion is done in another step that should not prevent publication: maybe can merge as is and figure out deletion issues later (that will still make the PR status fail until deletion problem is resolved)

**Review by balsarori — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (atchertchian) [reply .github/workflows/main.yml:385]: Added the step to pass that version in a file named "VERSION" for Makefile logic...

**Comment by atchertchian** [comment]
This PR was merged too soon -> will need to issue another PR to fix the logic, see https://github.com/Activiti/activiti-cloud-application/pull/695

**Pull request merged**